views/forms/ConfigForm/ConfigForm.css

```diff
@@ -0,0 +1,7 @@
+.ConfigFormActions {
+  position: fixed;
```

Comment:
Usually this is a red flag for me, but Material is weird. Is `position: fixed` recommended somewhere? What problem is this resolving?

Reply:
this keeps the add a configuration card visible on screen, there can be up anywhere from 1-400+ cards on screen. By having it visible as a user scrolls they can add more cards as they need

Reply:
So it is fixed relative to the whole page, no weird `position: relative` stuff going on? Okay, works for me.